mmpose/datasets/pipelines/loading.py

```diff
@@ -71,10 +71,10 @@ def __call__(self, results):
                 # The original results['img'] is assumed to be image(s) in BGR
                 # order, so we convert the color according to the arguments.
                 if results['img'].ndim == 3:
-                    results['img'] = mmcv.bgr2rgb(results['img'])
+                    results['img'] = results['img'][:, :, ::-1]
                 elif results['img'].ndim == 4:
                     results['img'] = np.concatenate(
-                        [mmcv.bgr2rgb(img) for img in results['img']], axis=0)
+                        [results['img'][:, :, ::-1] for img in results['img']], axis=0)
```

Comment:
should be `[img[:, :, ::-1] for img in results['img']]` ?

Reply:
I'll fix it

Reply:
I use `img[:, :, ::-1]` instead of `bgr2rgb(img)`, because the former is ligthweight for cpu.